### PR TITLE
OKTA-552301 Team Scotty: Update to rate limits dashboard

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added a note indicating warning, burst, and violation emails are not sent for every call.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-552301](https://oktainc.atlassian.net/browse/OKTA-552301)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 1]
```diff
@@ -111,6 +111,12 @@ For example, you may want to customize the rate limit threshold when an API norm
 
 Bursts, or unexpected API calls that hit and exceed the rate limit, generate a system log event, an entry in the Rate Limit Monitoring widget, and issue email notifications. Your org receives a notification when the default rate limit is hit. See [Burst rate limits](/docs/reference/rate-limits/#burst-rate-limits).
 
+#### Violation notification
+
+Rate limit violations generate a system log event, an entry in the Rate Limit Monitoring widget, and issue email notifications. Your org receives a notification when the default rate limit is hit. See [API rate limits and violations](#api-rate-limits-and-violations).
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:116]:
```suggestion
Rate limit violations generate system log events, entries in the Rate Limit Monitoring widget, and email notifications. Your org receives a notification when the default rate limit is hit. See [API rate limits and violations](#api-rate-limits-and-violations).
```

**Review by isaacdurham-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md](https://github.com/okta/okta-developer-docs/blob/2101e3ffa630b6b36bc8c0f8c99f216986e61fd6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-dashboard%2Findex.md) | 73 | 162 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4caaee46-5ebf-47f8-9124-ad37cdff5833) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4230dcaf-b391-4b61-91cc-1bb54d9cd43d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md](https://github.com/okta/okta-developer-docs/blob/539567b8eec8bf0eedc8198f625d01bfb84d363c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-dashboard%2Findex.md) | 76 | 133 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3d2a7d92-daf7-479e-b42e-22d01f2c390b) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.fb9b8001-fe37-4448-a301-75d0e8e95366?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**